### ISSUE TITLE
Best pretrained model

### ISSUE THREAD

**Issue by violetteshev** [airsplay/lxmert#83]
Hi, thanks for your work! I want to download the model that was pretrained for 12 epochs via this link:
```https://nlp1.cs.unc.edu/data/github_pretrain/lxmert/EpochXX_LXRT.pth```

I need the snapshot that has the lowest validation loss. Could you please tell me which epoch number it is?
